app/run.py

```diff
@@ -12,4 +12,5 @@
 
 app = create_app()
 
-app.run(debug=True, threaded=True, host='0.0.0.0', port=9000)
+print(parent_dir)
+#app.run(debug=True, threaded=True, host='0.0.0.0', port=9000)
```

Comment:
What's the reason for this being removed?

Reply:
shouldn't thanks.